lib/v2/penguin-random-house/router.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function (router) {
+    router.get('/the-read-down', require('./thereaddown'));
+    router.get('/articles', require('./articles'));
```

Comment:
```suggestion
    router.get('/articles', require('./articles'));
    router.get('/the-read-down', require('./thereaddown'));
```
Please sort it in ascending order